src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -55,9 +55,15 @@ public sealed override ImmutableArray<Symbol> GetMembers()
         {
             EnsureAllMembersLoaded();
 
-            var builder = ArrayBuilder<Symbol>.GetInstance();
-            builder.AddRange(GetMemberTypesPrivate());
-            builder.AddRange(GetMemberNamespacesPrivate());
+            var memberTypes = GetMemberTypesPrivate();
+            var builder = ArrayBuilder<Symbol>.GetInstance(memberTypes.Length + lazyNamespaces.Count);
+
+            builder.AddRange(memberTypes);
+            foreach (var pair in lazyNamespaces)
+            {
+                builder.Add(pair.Value);
+            }
```

Comment:
Why not `builder.AddRange(lazyNamespaces.Values);`?

Reply:
I'll add a comment in code. This is a very hot allocation path and the collection/enumerator were showing up in perf traces.

Reply:
I've benchmarked iterating the `Dictionary<TKey, TValue>` vs. `Dictionary<TKey, TValue>.Values` and second seems about 30% faster